src/app/redux/accountHistorySlice.ts

```diff
@@ -1,14 +1,110 @@
-import { PayloadAction, createSlice } from "@reduxjs/toolkit";
+import {
+  PayloadAction,
+  createSlice,
+  createAsyncThunk,
+  createSelector,
+} from "@reduxjs/toolkit";
+import { RootState, AppDispatch } from "./store";
 import * as adex from "alphadex-sdk-js";
+import { SdkResult } from "alphadex-sdk-js/lib/models/sdk-result";
+import { getRdt, RDT } from "../subscriptions";
 
+// TYPES AND INTERFACES
+export enum Tables {
+  OPEN_ORDERS = "Open Orders",
+  ORDER_HISTORY = "Order History",
+  TRADE_HISTORY = "Trade History",
+}
 export interface AccountHistoryState {
   trades: adex.Trade[];
+  orderHistory: adex.OrderReceipt[];
+  loading: boolean;
```

Comment:
Is it used somewhere? I only see that it's set and updated, maybe it's something left over from the previous versions and is not needed anymore?

Reply:
loading: boolean;  -> yes this is currently unused.  I kept it for now because im thinking if we might implement in the future for a transaction pop up loading block (for both cancel order and make an order)  or a loader we might be able to use that.
However, if you find it unnecessary then I would be happy to remove it from the code.  Let me know.

Reply:
Yes, let's remove and add when/if we use it.